packages/docs/src/pages/startup/examples.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Examples
+weight: 4
+---
+
+<h2 id="examples" class="ds-h2 ds-u-color--primary-darker">Examples</h2>
```

Comment:
The headings here are not done the same way as they are in other parts of the docs. I would propose removing all the top-level h2s entirely from these new pages, since they are duplicated in the main page title (since each section that used to be in the old single page Getting Started guide is now its own page.) This is what I mean visually with that, basically, the h2 is redundant here:

![screen shot 2019-01-17 at 3 42 47 pm](https://user-images.githubusercontent.com/3903208/51347608-97fa5180-1a6e-11e9-88b8-81cb04f6d92d.png)

I would also suggest maybe just using markdown syntax for all the headings. So `## Need Help` instead of the H2 tag below. I think that will standardize the colors with the rest of the docs (unless there is some reason I don't know about that the headings for these pages should be blue.)

Reply:
Removed the second, redundant header. I think the blue h2 was an attempt to hang on to typographical styles, but that doesn't mean we need to continue that.